src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -418,6 +418,37 @@ public FunctionDescriptor GetFunctionDescriptor(string managledName)
         }
 
         /// <summary>
+        ///     Returns a list of function descriptors associated with the function name.
+        /// </summary>
+        /// <param name="mangledName"></param>
+        /// <returns></returns>
+        public IEnumerable<FunctionDescriptor> GetAllFunctionDescriptors(string mangledName)
+        {
+            IEnumerable<FunctionDescriptor> descriptors = null;
+            string qualifiedName = mangledName.Split('@')[0];
```

Comment:
Why are you doing the split twice, once outside this function call and once here?